arches/app/utils/betterJSONSerializer.py

```diff
@@ -1,19 +1,16 @@
 import datetime
 import decimal
-import types
```

Comment:
I could revert this to get the python test coverage to pass, but really we should just #11909